src/System.Private.ServiceModel/tests/Scenarios/Binding/Tcp/NetTcpBindingTests.cs

```diff
@@ -12,15 +12,28 @@ public partial class Binding_Tcp_NetTcpBindingTests : ConditionalWcfTest
     // Default settings are:
     //                         - SecurityMode = Transport
     //                         - ClientCredentialType = Windows
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+    [ActiveIssue(832)] // Windows Stream Security is not supported in NET Native
+#else
     [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Windows_Authentication_Available))]
-#if !FEATURE_NETNATIVE
     [ActiveIssue(945, PlatformID.AnyUnix)] // NegotiateStream works on Windows and Linux, but we have not yet automated ambient credential configuration in Linux
```

Comment:
We should remove this AcitveIssue 945 and the Root_Certificate_Installed condition from this test. The Windows_Authentication_Available condition is supposed to be sufficient to skip the test when requirement is not met.